Content.Shared/Preferences/HumanoidCharacterProfile.cs

```diff
@@ -272,7 +272,9 @@ public HumanoidCharacterProfile WithAntagPreference(string antagId, bool pref)
         }
 
         public string Summary =>
-            Loc.GetString("{0}, {1} years old human. {2:Their} pronouns are {2:they}/{2:them}.", Name, Age, this);
+            Loc.GetString(" This is {0}. {3:They} {3:are} a {2} year old human {1}.", Name, Sex.ToString().ToLower(), Age, this);
```

Comment:
You ever try out the new string interprolation?


$" This is {name}. {this:They} {this: are} a {Age} year old human {Sex.ToString().ToLower()}."

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Reply:
I think there's an internal Loc reason for doing it that way